src/guides/anatomy-of-an-http-transaction.md

```diff
@@ -0,0 +1,360 @@
+# Anatomy of an HTTP Request
+
+The purpose of this guide is to impart a solid understanding of the process of
+Node.js HTTP handling. We'll assume that you know, in a general sense, how HTTP
+requests work, regardless of language or programming environment. We'll also
+assume a bit of familiarity with Node.js [`EventEmitters`][] and [`Streams`][].
+If you're not quite familiar with them, it's worth taking a quick read through
+the API docs for each of those.
+
+## Create the Server
+
+Any node web server application will at some point have to create a web server
+object. This is done by using [`createServer`][].
+
+```javascript
+var http = require('http');
+
+var server = http.createServer(function(request, response) {
+  // magic happens here!
+});
+```
+
+The function that's passed in to [`createServer`][] is called once for every HTTP
+request that's made against that server, so it's called the request handler. In
+fact, the [`Server`][] object returned by [`createServer`][] is also an
+[`EventEmitter`][], and what we have here is just shorthand for creating a
+`server` object and then adding the listener later.
+
+```javascript
+var server = http.createServer();
+server.on('request', function(request, response) {
+  // the same kind of magic happens here!
+});
+```
+
+When an HTTP request hits the server, node calls the request handler function
+with a few handy object for dealing with the transaction, `request` and
+`response`. We'll get to those shortly.
+
+In order to have this thing actually serving requests, the [`listen`][] method
+needs to be called on the `server` object. In most cases, all you'll need to put
```

Comment:
This sentence seems a little clunky. Instead of "all you'll need to put for arguments is the port number" perhaps "all you'll need for arguments is the port number". Also the "you want the server to listen on" sub-clause makes the it quite a long sentence. Perhaps split the second have starting with "but there are some other options..." into a separate sentence.